index3.html

```diff
@@ -0,0 +1,85 @@
+          <div class="flex-main main__div">
+            <div>
+              <h1 class="main__div__h1">Chocolate Pizza</h1>
+              <h3 class="main__div__h3">POSTED ON 15 DEC 2013 /DESSERTS</h3>
+            </div>
+            <p class="print"> <img src="print-icon.png"> PRINT</p>
+          </div>
+          <img class="main__img" src="choco-pizza.png">
+          <p class="main__p"> For the fig-swirl: Melt butter over medium heat in a saucepan. Add brown sugar and stir to dissolve. Halve all of the figs and toss in the saucepan with water and lemon juice. Cook over medium heat, stirring frequently until you have a chunky-jammy mixture. Add salt with one or two stirs, set aside and let cool completely.</p>
+          <p class="main__p">Ice cream: In a small pot  over medium heat, combine milk, and granulated sugar until sugar is completely dissolved and milk is just barely lukewarm. Whisk in the egg yolks. Set mixture in the fridge and wait until the fig mixture is cooled.</p>
+          <p class="main__p">Using and ice cream machine, pour liquids into the frozen basin and process according to manufacturer instructions, i.e., let spin and thicken for 20 minutes before adding mascarpone, fig jam mixture, and the nuts. Continue to process for +/- 10 minutes. Pour semi-frozen mixture into a Pyrex dish or glass Tupperware. Freeze for at least two hours before serving.
+          </p>
+          <section class="flex-recipeList">
+            <div>
+              <label>
+                <ul>
+                  <li class="listItem"><input type="checkbox">1 1/2 cups milk</input></li>
+                  <li class="listItem"><input type="checkbox">1/2 cup mascarpone</input></li>
+                  <li class="listItem"><input type="checkbox">1/2 tsp pink salt</input></li>
+                  <li class="listItem"><input type="checkbox">1lb Black Mission Figs</li>
+                  <li class="listItem"><input type="checkbox">1/2 cup brown sugar</input></li>
+                  <li class="listItem"><input type="checkbox">2-4 tbsp water</input></li>
+                </ul>
+              </label>
+            </div>
+            <div >
+              <label>
+                <ul>
+                  <li class="listItem"><input type="checkbox">1 1/2 cups heavy cream</input></li>
+                  <li class="listItem"><input type="checkbox">1/3 cup granulated sugar</input></li>
+                  <li class="listItem"><input type="checkbox">2 egg yolks</input></li>
+                  <li class="listItem"><input type="checkbox">1 lemon, juiced</input></li>
+                  <li class="listItem"><input type="checkbox">2 tbsp butter</input></li>
+                  <li class="listItem"><input type="checkbox">1 cup honey roasted pecans, roughly chopped</input></li>
+                </ul>
+              </label>
+            </div>
+          </section>
+          <section class="flex-bio main__section-bio">
+            <img src="van-pic.png">
```

Comment:
I mentioned it before, but this image since it's in a flex container is in danger of being smooshed. Try to put a width of 100% on it.